commit d48a35c1ef58b3615b2285916fa8f9740cdda748 (refs/changes/48/2748/1)
Author: Bryan Cantrill <bryan@joyent.com>
Date:   2017-10-08T08:58:51+00:00 (2 years ago)
    
    OS-6385 zfs_arc_overflow_shift needs sufficient margin for ARC reclaim

diff --git a/usr/src/uts/common/fs/zfs/arc.c b/usr/src/uts/common/fs/zfs/arc.c
index 5b58379915..fe60c09daa 100644
--- a/usr/src/uts/common/fs/zfs/arc.c
+++ b/usr/src/uts/common/fs/zfs/arc.c
@@ -303,7 +303,7 @@ int zfs_arc_evict_batch_limit = 10;
 static int		arc_grow_retry = 60;
 
 /* shift of arc_c for calculating overflow limit in arc_get_data_impl */
-int		zfs_arc_overflow_shift = 8;
+int		zfs_arc_overflow_shift = 3;
 
 /* shift of arc_c for calculating both min and max arc_p */
 static int		arc_p_min_shift = 4;
